tools/haskell-translator/lhs_pars.py

```diff
@@ -706,7 +710,8 @@ def typename_transform(line, header, d):
         [oldtype] = line.split()
     except:
         if header not in known_type_assignments:
-            warning('type assignment with parameters not supported %s' % d.body, filename)
+            warning('type assignment with parameters not supported %s, (%s)' %
+                    (d.body, header), filename)
```

Comment:
not following, what do you want printed here?

Reply:
`header` is the thing it's called on, which turned out to be more useful for debugging than just `d.body` which it printed before.